deepchecks/utils/performance/error_model.py

```diff
@@ -179,7 +179,9 @@ def error_model_display(error_fi: pd.Series,
 
             # Partition data into two groups - weak and ok:
 
-            in_segment_indicis = cum_sum_ratio <= min_segment_size
+            # The weak segment is all the weakest categories accumulated together until they comprise at least
+            # min_segment_size of the total data.
+            in_segment_indicis = np.arange((cum_sum_ratio >= min_segment_size)[0])
```

Comment:
`(cum_sum_ratio >= min_segment_size)` creates boolean array.
`(cum_sum_ratio >= min_segment_size)[0]` returns the first boolean in the array. 
what am I missing?

Reply:
Instead of creating an array of all categories whose cumulative count is above min_segment_size (which, if the first is above min_segment_size, will cause all the data to be in the second segment), we create an array of all categories **up to** the first category whose cumulative count is above min_segment_size, thus splitting the data into a segment of "all the weakest categories taking at least min_segment_size of the data" and "all the rest".

Reply:
I'm with Matan. I still don't understand why you take the first item in the array and feed it to np.arange.

Reply:
Y'all were right, fixed